core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/AbstractContractNegotiationManager.java

```diff
@@ -0,0 +1,120 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
same here, is this change related to this PR ? Is it due to some merge issue ?

Looks like this PR has many changes like this.